clients/include/testing_tpmv_strided_batched.hpp

```diff
@@ -0,0 +1,258 @@
+/* ************************************************************************
+ * Copyright 2018-2020 Advanced Micro Devices, Inc.
+ * ************************************************************************ */
+
+#include "cblas_interface.hpp"
+#include "flops.hpp"
+#include "near.hpp"
+#include "norm.hpp"
+#include "rocblas.hpp"
+#include "rocblas_datatype2string.hpp"
+#include "rocblas_init.hpp"
+#include "rocblas_math.hpp"
+#include "rocblas_random.hpp"
+#include "rocblas_test.hpp"
+#include "rocblas_vector.hpp"
+#include "unit.hpp"
+#include "utility.hpp"
+
+template <typename T>
+void testing_tpmv_strided_batched_bad_arg(const Arguments& arg)
+{
+    const rocblas_int       M           = 100;
+    const rocblas_int       incx        = 1;
+    const rocblas_int       batch_count = 1;
+    const rocblas_stride    stride_a    = (M * (M + 1)) / 2;
+    const rocblas_stride    stride_x    = M;
+    const rocblas_operation transA      = rocblas_operation_none;
+    const rocblas_fill      uplo        = rocblas_fill_lower;
+    const rocblas_diagonal  diag        = rocblas_diagonal_non_unit;
+
+    rocblas_local_handle handle;
+
+    size_t size_A = (M * (M + 1)) / 2;
+
+    host_strided_batch_vector<T> hA(size_A, 1, stride_a, batch_count);
+    CHECK_HIP_ERROR(hA.memcheck());
+
+    host_strided_batch_vector<T> hx(M, incx, stride_x, batch_count);
+    CHECK_HIP_ERROR(hx.memcheck());
+
+    device_strided_batch_vector<T> dA(size_A, 1, stride_a, batch_count);
+    CHECK_HIP_ERROR(dA.memcheck());
+
+    device_strided_batch_vector<T> dx(M, incx, stride_x, batch_count);
+    CHECK_HIP_ERROR(dx.memcheck());
+
+    //
+    // Initialize.
+    //
+    rocblas_init(hA, true);
+    rocblas_init(hx);
+
+    //
+    // Transfer.
+    //
+    CHECK_HIP_ERROR(dA.transfer_from(hA));
+    CHECK_HIP_ERROR(dx.transfer_from(hx));
```

Comment:
Leaving the memory uninitialized I think is fine.

Reply:
Which uninitialized memory?

Reply:
I'm saying I wouldn't bother with rocblas_init or upload.  This is inside bad_arg (look at larger code context), based on the parameters you pass the tests never use hA, hx, dA, dx values but if they do uninitialized memory is fine.

Reply:
done.